snuba/subscriptions/data.py

```diff
@@ -172,14 +173,15 @@ def build_request(
     def from_dict(
         cls, data: Mapping[str, Any], entity_key: EntityKey
     ) -> SubscriptionData:
-        entity_subscription = get_entity_subscription(entity_key)(data_dict=data)
+        entity_subscription = get_entity_subscription(entity_key)()
 
         return SubscriptionData(
             project_id=data["project_id"],
             # Always cast to int in case any old subscriptions are stored as float
             time_window_sec=int(data["time_window"]),
             resolution_sec=int(data["resolution"]),
             query=data["query"],
+            organization=data.get("organization", None),
```

Comment:
Ideally this would be like:
```suggestion
            organization=data["organization"],
```

Curious why you decided to provide a fallback here? Was it to deal with deployment issues where some older data from before this change gets deployed won't have the "organization" key yet?

Reply:
Yes, it is for lagging subscriptions that might not have `organization` key.

Reply:
I don't know that `organization` is passed for all subscriptions. It's only required by sessions, so I don't think we can assume there will be an organization for transactions subscriptions for example.